src/cdk/accordion/accordion-item.ts

```diff
@@ -125,4 +134,13 @@ export class CdkAccordionItem implements OnDestroy {
       this.expanded = true;
     }
   }
+
+  private _subscribeToExpandCollapseAllActions(): Subscription {
```

Comment:
This method should likely be called `subscribetoOpenCloseAllActions()`

Reply:
Done.